### PR TITLE
FFS-2108 - added email reminder es translations

### PR DESCRIPTION
**Pull request by GeorgeCodes19**
## Ticket

Resolves [FFS-2108](https://jiraent.cms.gov/browse/FFS-2108)

## Changes

- Added `es` i18n entries for email reminders

### CODE REVIEW

File: app/config/locales/es.yml [hunk 1]
```diff
@@ -22,6 +22,23 @@ es:
...
+      feedback_button: Comparta tus comentarios
+      feedback_header: "¿Tiene algún comentario sobre el programa piloto de SNAP?"
+      footer: Este es un mensaje generado automáticamente de %{agency_acronym}. No responda a este correo electrónico.
+      greeting: Hola,
+      instructions: Inicia sesión en su cuenta de proveedor de nóminas (por ejemplo, ADP, Lyft o Amazon Flex) para compartir sus datos de ingresos directamente con %{agency_acronym}. Si usted no cumple con la fecha límite, este enlace expirará y tendrá que solicitar una nueva invitación.
+      main_action_html:
+        default: "<strong>Verify your income</strong> as part of your SNAP recertification by <strong>%{deadline}</strong>."
+        ma: "<strong>Verify your income</strong> as part of your SNAP recertification by <strong>%{deadline}</strong>."
+        nyc: "<strong>Verify your income</strong> as part of your SNAP application or recertification by <strong>%{deadline}</strong>."
+        sandbox: "<strong>Verify your income</strong> as part of your SNAP recertification by <strong>%{deadline}</strong>."
```
Comment (tdooner) [inline app/config/locales/es.yml:35]: Did we get these ones translated?

Comment (GeorgeCodes19) [reply app/config/locales/es.yml:35]: Whoops, I think I put them in the wrong file! Good eye

**Review by tdooner — APPROVED** [review]
Just curious - have you been using your script for this, or just doing it manually?

**Comment by GeorgeCodes19** [comment]
> Just curious - have you been using your script for this, or just doing it manually?

At first I used the script that I created in a previous PR that included all of these translations. These were out of scope for that ticket though. Since this file was already in my Rubymine file history from running my script the first around- I just popped open the file history and clicked `>>` to move them over.

**Pull request merged**